### DISCUSSION TITLE
How to fix this error on lazer?

### DISCUSSION THREAD

**Discussion by Iva120** [ppy/osu#20885] [Q&A] [answered]
I updated osu!lazer yesterday. After i did, all my custom gamemodes were nowhere to be seen in-game, even though they were still in the files. How do i fix this?
![osu_2022-10-23_14-00-46](https://user-images.githubusercontent.com/108094045/197388528-aa41cf65-fe51-44bb-ad06-e586bdee35bb.jpg)

![Screenshot (14)](https://user-images.githubusercontent.com/108094045/197388625-c1c1115f-185a-4cd5-9cb7-ff97b8c713da.png)

**Comment by nagi-desuuu** [comment]
Have you tried upgrading those rulesets from their respective developers? And did you put them in the rulesets folder just to be sure?

**Comment by peppy** [comment, marked as answer]
The screenshot shows the rulesets being in the wrong folder (they are in the folder for the previous release). As mentioned above, you should put them in the `rulesets` folder. You can access it by clicking "open osu! folder" in the game settings

**Reply by Iva120** [reply]
It worked. But weird thing was i'd initially put them in that folder and they still showed up in-game, but also now some of the can't bee seen in game. I'll send a screenshot of the errors bellow. Prob. just need to download the latest version of the rulesets tho, idk.
![osu_2022-10-23_20-42-51](https://user-images.githubusercontent.com/108094045/197407451-3a732902-8988-43fb-bcb3-9058f651dbc4.jpg)

**Reply by peppy** [reply]
You will need to find a new version of the ruleset (or in the case that doesn't exist, contact the developers), as per the message you're seeing on your screen. They are out of date.
